File: Modul-2/Python/PRAK204-2310817320007-Zahra Nabila.py
phi = 3.142857
Nilai = input() . split()

if len (Nilai)== 1:
    r = float(Nilai[0])
    t = float(input())

elif len(Nilai)== 2:
    r, t = map(float, Nilai)

Volume = phi * r * r * t
Luas = 2 * phi * r * ( r + t)
Keliling = 2 * phi * r

print(f"Volume      = {Volume:.2f}")
print(f"Luas        = {Luas:.2f}")
print(f"Keliling    = {Keliling:.2f}")

File: Modul-2/C/PRAK204-2310817320007-ZahraNabila.c
#include <stdio.h>
#define phi 3.142857

int main()
{
    float r, t, Volume, Luas, Keliling;

    scanf("%f %f", &r, &t);

    Volume      = phi*r*r*t;
    Luas        = 2*phi*r*(r + t);
    Keliling    = 2*phi*r;

    printf("Volume      = %.2f\n", Volume);
    printf("Luas        = %.2f\n", Luas);
    printf("Keliling    = %.2f\n", Keliling);

    return 0;
}